index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="section-title">I'm Anderson Glad to see you!</h1>
     </section>
     <section id="works-section">
       <div class="work-container">
-        <img class="align-item" src="./assets/images/work-1.png" alt="Tonic">
+        <img class="align-item card-image" src="./assets/images/work-1.png" alt="Tonic">
         <div class="work-aditional-info">
```

Comment:
- [ ] Kindly align the about section of the page with the projects card as per the Figma design. ✍️

- **Your Design**

![Screenshot from 2022-12-10 08-02-09](https://user-images.githubusercontent.com/17243325/206831144-0002dd5d-9351-420c-8a6a-d6e9caf5694f.png)

- **Figma Design**

![Screenshot from 2022-12-10 08-01-01](https://user-images.githubusercontent.com/17243325/206831201-95d39770-db20-4f5b-8e53-b88e34a6f11a.png)

Reply:
Thanks for the comment. I will fix it to accomplish the template